pycloudlib/cloud.py

```diff
@@ -260,6 +260,10 @@ def use_key(self, public_key_path, private_key_path=None, name=None):
         self._log.debug("using SSH key from %s", public_key_path)
         self.key_pair = KeyPair(public_key_path, private_key_path, name)
 
+    def version(self) -> Optional[str]:
+        """Version string of the platform."""
+        return None
```

Comment:
Is there a lot of value in us returning None here instead of just an empty string ""? We can then avoid the Optional[str] handling for typing validation.